WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/TotalStatsMapper.kt

```diff
@@ -23,7 +23,13 @@ class TotalStatsMapper @Inject constructor(
         val previousWeekLikes = getPreviousWeekDays(dates, LIKES)
         val positive = currentWeekLikes.sum() >= previousWeekLikes.sum()
 
-        return ValueWithChartItem(currentWeekSumFormatted, currentWeekLikes, positive)
+        return ValueWithChartItem(
+                value = currentWeekSumFormatted,
+                chartValues = currentWeekLikes,
+                positive = positive,
+                // Add extra bottom margin if there is no information text.
```

Comment:
Have you tried using [layout_goneMarginBottom](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout) instead in the layout

Reply:
I thought that, I didn't like adding margin paramater too but we're adding information text or not adding it in the use case. To use 'layout_goneMarginBottom', we need to add information text always, and then make it gone. I thought it could be hard to understand this logic later, but I can write a comment to explain.

Thanks for the suggestion. Do you still think 'layout_goneMarginBottom' is better?

Reply:
How about adding more bottom margin to value and line chart layout?

Reply:
We can't add bottom margin in all case. Total cards have 3 rows, like (`8dp margin` + `1. row 48dp` + `2. row 48dp` + `3. row 48dp` + `8dp margin`). If 3. row doesn't exist, we should add the bottom margin to the 2. row.

Actually, these margins are vertical padding of the card view. But we can't add vertical padding to the card view, because not all cards have that vertical padding, eg. "+ Add a new stats card" card

<img width="360" alt="image" src="https://user-images.githubusercontent.com/2471769/171591950-6fb44d7b-1e8b-42ee-a2db-9514daf5d74c.png">